libs/core/.stylelintrc.json

```diff
@@ -26,6 +26,7 @@
           "custom-elements"
         ]
       }
-    ]
+    ],
+    "max-nesting-depth": 3
```

Comment:
Same comment as above

Reply:
Nesting in our CSS, [as seen here](https://github.com/Kajabi/sage/pull/15/files#diff-aef3889db0e6b395692b418b189dc651d500646a8d7ef82e7ee74b5f671e65c6R78-R97), is used to scope logic into a modifier class. I find this the easiest way to parse the code but if others have thoughts on this and would prefer to write it out all in one selector with less nesting I'd be open to that. Curious what the team thinks!

Reply:
I think this warrants a broader discussion on how we want to organize our CSS. Because we have the `:host` selector to use, I imagine we could leverage this to eliminate the need for nesting or nesting too deeply.